src/ast/astExtractor/block-statement-extractor.ts

```diff
@@ -1,38 +1,42 @@
+import { ExpressionExtractor } from "./expression-extractor";
+import { Identifier, UnannType } from "../types/classes";
+import { StatementExtractor } from "./statement-extractor";
+import { TypeExtractor } from "./type-extractor";
```

Comment:
Why rearranged these imports up here? I thought they were alr in the correct positions?